### DISCUSSION TITLE
[Question] performance comparison with rulid.rs

### DISCUSSION THREAD

**Discussion by ngortheone** [dylanhart/ulid-rs#42] [Q&A]
Were there any performance comparisons between this crate and https://github.com/mmacedoeu/rulid.rs?

If no will you be interested in some?

**Comment by dylanhart** [comment]
The other feature-complete implementation of ulid that I know about is [rusty_ulid]. rulid.rs looks to be more a transcription of the original JavaScript implementation into Rust.

If you want to do this research I'd think that testing [rusty_ulid] would be best. That being said, I don't expect there to be major performance difference between the two libraries. This library is extremely similar to rusty_ulid and vice-versa.

[rusty_ulid]: https://github.com/huxi/rusty_ulid

**Comment by dylanhart** [comment]
I found the following which may be interesting to you: https://github.com/j5ik2o/ulid-generator-rs#benchmarks
